ui/src/pages/__tests__/SearchPageVulns.test.tsx

```diff
@@ -1707,7 +1707,7 @@ describe("SearchPage component", () => {
 					within(dialog).getByText("Aqua CLI Scanner (Docker)");
 					within(dialog).getByText("NPM Audit (NodeJS)");
 					within(dialog).getByText("Snyk");
-					within(dialog).getByText("Trivy (Container Images)");
+					within(dialog).getByText("Trivy Container Image");
```

Comment:
you may want to add one more test here to ensure that the new SCA plugin appears in this field as well:
```
within(dialog).getByText("Trivy SCA");
```

Reply:
Disregard, looks like this is actually from mock data